### ISSUE TITLE
double printing of warnings and info

### ISSUE THREAD

**Issue by CarloLucibello** [guildai/guildai#181]
I'm running a pytorch-lightning script, which makes use of some kind of warning/info output. 
When running the standalone script I get the output 
```
GPU available: False, used: False
No environment variable for node rank defined. Set as 0.
LR MileStones [50, 75]

  | Name         | Type             | Params
----------------------------------------------
0 | net          | LeNet            | 431 K 
1 | net.conv1    | Conv2d           | 520   
2 | net.max_pool | MaxPool2d        | 0     
3 | net.conv2    | Conv2d           | 25 K  
4 | net.fc1      | Linear           | 400 K 
5 | net.fc2      | Linear           | 5 K   
6 | net.relu     | ReLU             | 0     
7 | loss         | CrossEntropyLoss | 0     
```
Running the scipt with Guild instead, every output is duplicated:
```
GPU available: False, used: False
INFO: [lightning] GPU available: False, used: False
No environment variable for node rank defined. Set as 0.
WARNING: [lightning] No environment variable for node rank defined. Set as 0.
LR MileStones [50, 75]

  | Name         | Type             | Params
----------------------------------------------
0 | net          | LeNet            | 431 K 
1 | net.conv1    | Conv2d           | 520   
2 | net.max_pool | MaxPool2d        | 0     
3 | net.conv2    | Conv2d           | 25 K  
4 | net.fc1      | Linear           | 400 K 
5 | net.fc2      | Linear           | 5 K   
6 | net.relu     | ReLU             | 0     
7 | loss         | CrossEntropyLoss | 0     
INFO: [lightning] 
  | Name         | Type             | Params
----------------------------------------------
0 | net          | LeNet            | 431 K 
1 | net.conv1    | Conv2d           | 520   
2 | net.max_pool | MaxPool2d        | 0     
3 | net.conv2    | Conv2d           | 25 K  
4 | net.fc1      | Linear           | 400 K 
5 | net.fc2      | Linear           | 5 K   
6 | net.relu     | ReLU             | 0     
7 | loss         | CrossEntropyLoss | 0     
```

**Comment by gar1t** [comment]
Thanks for the report! Do you have a simple code example that you can provide that demonstrates this behavior?

**Comment by CarloLucibello** [comment]
here is a MWE, reduced from pytorch-lightning [code](https://github.com/PyTorchLightning/pytorch-lightning/blob/4cdebf9a6401e47ad99745856a4e009537bee494/pytorch_lightning/__init__.py)

```python
import logging 
log = logging.getLogger("ciao")
log.addHandler(logging.StreamHandler())
log.setLevel(logging.INFO)
log.info("aaa")
log.warning("bbbb")
```
with Guild run output
```
aaa
INFO: [ciao] aaa
bbbb
WARNING: [ciao] bbbb
```
 Without the StreamHandler, there is no output in the python script and only the INFO and WARNING lines in the Guild run

**Comment by gar1t** [comment]
I added an issue doc for this:

https://github.com/guildai/issue-resolution/tree/master/181-double-printing-of-warnings-and-info

TLDR; this is expected behavior, though I understand that it's surprising. Guild's log init occurs before your code is run. You can disable this by setting the environment variable `LOG_INIT_SKIP` to `1`. This has to be done in the Guild file. The doc above provides examples.

**Comment by gar1t** [comment]
Dup of #55
